FEATURE-LIFECYCLE-CHECKLIST.md

```diff
@@ -13,58 +13,64 @@ be labeled as Alpha, Beta, or Stable. This is a shortened version of the full
 A development feature can be officially labeled as Alpha once it meets the following
 requirements:
 
-* *Config*: Requires explicit user action to enable (e.g. a config field, config resource, or installation action).
+* Config
+    * Requires explicit user action to enable (e.g. a config field, config resource, or installation action).
 
-* *Docs*: Reference docs are published on istio.io.
+* Docs
+    * Reference docs are published on istio.io.
+    * Basic feature docs are published on istio.io describing what the feature does, how to use it, and any caveats.
+    * A reference to the design doc / issue is published on istio.io.
 
-* *Docs*: Basic feature docs are published on istio.io describing what the feature does, how to use it, and any caveats.
+* Tests
+    * Test coverage is 90%.
```

Comment:
Is this 90% still accurate for Alpha?  I recall a discussion of 70% for alpha in TOC mtg mins somwhere.

Reply:
Martin proposed alpha 60, beta 70, stable 90. Maybe change to alpha 70, beta 80, stable 90?  :)

Reply:
Changed the numbers.